backend/de.metas.edi/src/main/sql/postgresql/system/80-de.metas.edi/5726254_sys_gh18259_add_M_InOut_DesadvLine_V.sql

```diff
@@ -0,0 +1,64 @@
+drop view if exists M_InOut_DesadvLine_V
+;
+
+create or replace view M_InOut_DesadvLine_V as
+select shipment.m_inout_id                                                       as M_InOut_Desadv_ID,
+       case
+           when desadvInOutLine.edi_desadvline_id > 0 then
+               desadvInOutLine.edi_desadvline_id || '-' || desadvInOutLine.m_inoutline_id
+                                                      else
+               uuid_generate_v4() || ''
```

Comment:
seems as if this is different each time, so it probably won't work. 
when could `desadvInOutLine.edi_desadvline_id` be `<0` or `null` anyways

Reply:
actually, it works just fine :) this is about those desadv lines that are not part of the current shipment, so we don't want to capture the packs for them

Reply:
ah, ok, please add a comment to that effect to the DDL then